### PR TITLE
Changing cadence for dbt models hourly runs

### PR DESCRIPTION
**Pull request by sri-byte**
Moved a lot of stuff from hourly to nightly

#### Ticket Link

Fixes https://mattermost.atlassian.net/browse/MM-52375

### CODE REVIEW

**Comment by catalintomai** [comment]
As discussed, let's have this ticket addressing exclusively on the cadence changes and leave the the "updating the warehouse used" to a different PR associated to [MM-52376](https://mattermost.atlassian.net/browse/MM-52376)

**Comment by ifoukarakis** [comment]
@sri-byte can you please add the reasoning behind choices in the issue's description? A few patterns seem to exist, but I'm only guessing here. 

Tests are also failing. Let's fix them before merging.

**Comment by ifoukarakis** [comment]
@sri-byte CI/CD fails because source branch is on a different repo. Most likely you created a fork of the project in your personal account and then created a PR from that repo to this one. CI/CD is not supporting this type of PRs. Please close this PR and create a new one where both branches are in `mattermost/mattermost-data-warehouse` repo.

File: transform/snowflake-dbt/models/incident_collaboration/incident_daily_details.sql [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {{config({
     "materialized": "incremental",
     "schema": "incident_collaboration",
-    "tags":"hourly",
+    "tags":"nightly",
     "unique_key":"id",
-    "warehouse":"ANALYST_XS"
+    "warehouse":"TRANSFORM_XS"
```
Comment (catalintomai) [inline transform/snowflake-dbt/models/incident_collaboration/incident_daily_details.sql:6]: Let's revert this change (and similar ones) and move it to the PR addressing [MM-52376](https://mattermost.atlassian.net/browse/MM-52376)

**Review by catalintomai — APPROVED** [review]
